src/TheClassLessTheme/Recipes/TheClassLessTheme.Tests.recipe.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "TheClassLessTheme.Tests",
+  "displayName": "The ClassLess Theme Tests",
+  "description": "Provides a ClassLess theme for tests.",
+  "author": "Antoine Griffard",
+  "website": "http://antoinegriffard.com",
+  "version": "1.0.0",
+  "issetuprecipe": true,
+  "categories": [ "developer" ],
+  "tags": [ "ClassLess" ],
+
+  "steps": [
+    {
+      "name": "recipes",
+      "Values": [
+        {
+          "executionid": "TheClassLessTheme",
+          "name": "TheClassLessTheme"
+        }
+      ]
+    },
+    {
+      "name": "settings",
+      "UseCdn": false
```

Comment:
Ez itt meg a többi témánál önmagában biztos megoldja, hogy ne töltsön be webfontokat?

Reply:
Ahogy tesztelgettem nem, viszont nem tudom hogy akkor mi lenne a megoldás. Hasonlóra volt már példa valahol? Mert nézegettem más projekteket, de ilyesmit nem találtam. Mivel minden le van töltve még azt lehet csinálni, hogy kitörlöm a font linkeket.

Reply:
Hmm, hát ezzel most tovább kell az egészet bonyolítanunk, ne haragudj, de ezt nem lehetett előre látni.

Talán a legegyszerűbb, ha van külön egy külön local fontos CSS fájl, ami deklatárlja a fontokat. Azaz ez váltja ki a `@import url('https://fonts.googleapis.com/css2?family=Lora&family=Archivo&display=swap');` és hasonlókat (ezt már megcsináltad ugye). Utána pedig a cshtml-ben a `<style>`-ban megadhatod az `src` mellett a `cdnsrc`-t, ami a két változatra szól (lásd `StyleTagHelper`). Ezek a configtól függően fogják betölteni a megfelelő változatot.

Reply:
Kicseréltem a linkeket `style`-ra, és töröltem a lokális CSS-ből a linkeket. 

```
{% style src:"~/TheClassLessTheme/css/themes.css",
                cdn_src:"https://classless.de/classless.css"%}
```

Ez valid, viszont azzal van a probléma hogy a `cdn_src`-t nem használja akkor se, ha a recepibe beleírom hogy `"UseCdn": true`, és még a `style`-ba is beírom attribútumként hogy `use_cdn:"true"`.  Tehát mindig a localt használja. Ez ugye tesztnél elvárt, de ha nem tesztben vagyok, akkor is azt használja mindenképpen. Próbáltam release mode-ban buildelni, a settings-ben átállítani dolgokat pl. caching-et kikapcsolni, de akkor is a lokális CSS-t használja. Ezt nem tudom mi okozza, illetve hogy mennyire baj.

(Aztán ha ez megvan, akkor ugyanezt el kell játszanom a TheResumeTheme/BootstrapTheme-nél is, bár ott egyszerűbb a helyzet, mert csak kevesebb nem local hivatkozás van, viszont ott is le kell töltenem a fontokat.)

Reply:
Lásd [Teams](https://teams.microsoft.com/l/message/19:525a1a3656c845ae92c5e2c0d3617cc7@thread.skype/1644428627970?tenantId=959ede5a-38e9-44e6-adc8-85553beb8220&groupId=d6a5d775-6fb9-41dc-a7bb-481d4f884046&parentMessageId=1644428627970&teamName=Lombiq%20Developers&channelName=General&createdTime=1644428627970).